package.json

```diff
@@ -23,7 +23,7 @@
     "check-for-leaks": "^1.2.1",
     "colors": "^1.1.2",
     "dotenv-safe": "^4.0.4",
-    "dugite": "^1.45.0",
+    "dugite": "^1.87.0",
```

Comment:
Why bump this?

Reply:
this was done automatically when I ran tests locally with 
npm run test -- -g=preconnect,
I can revert this if needed

Reply:
Yeah please revert 👍

Reply:
done